packages/react/src/time-input/TimeInput.test.tsx

```diff
@@ -0,0 +1,132 @@
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import 'html-validate/jest';
+import { useState } from 'react';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { cleanup, render, screen } from '../../../../test-utils';
+import { TimeInput, type TimeInputProps, type TimeValue } from './TimeInput';
+
+expect.extend(toHaveNoViolations);
+
+afterEach(cleanup);
+
+type ControlledTimeInputProps = Omit<TimeInputProps, 'label'>;
+
+function renderTimeInput(props?: ControlledTimeInputProps) {
+	return render(<ControlledTimeInput {...props} />);
+}
+
+function ControlledTimeInput({
+	onChange: onChangeProp,
+	value: valueProp,
+	...props
+}: ControlledTimeInputProps) {
+	const [value, setValue] = useState<TimeValue | undefined>(valueProp);
+
+	function onChange(value: TimeValue) {
+		setValue(value);
+		onChangeProp?.(value);
+	}
+
+	return (
+		<TimeInput label="Time" onChange={onChange} value={value} {...props} />
+	);
+}
+
+async function getInput() {
+	const el = await screen.getByRole('textbox');
+	expect(el).toBeInstanceOf(HTMLInputElement);
+	expect(el).toBeInTheDocument();
```

Comment:
Again, I may have missed something here but I don't think this assertion is needed. If you can get the element on line 37, then it's in the document, right? Maybe there's more value in `toBeVisible()` but my gut feeling is this assertion isn't needed. Testing library / Jest will throw if it can't get the element.

Reply:
Agreed. Removing